lib/application.js

```diff
@@ -388,10 +388,10 @@ Origin.prototype.startServer = function () {
       callback();
     },
     function(callback) {
-      if (true === configuration.getConfig('isTestEnvironment')) {
+      if (true === configuration.getConfig('isTestEnvironment') || options.skipVersionCheck) {
         return callback();
       }
-
+     
```

Comment:
if you enabled .editorconfig in your editor and merge the develop branch, you'll not add this kind of whitespace ;)